### PR TITLE
Update group5.md

### PR DESCRIPTION
**Pull request by JackLaird0**

### CODE REVIEW

**Comment by JackLaird0** [comment]
@emhickmann

File: m4_sessions/1806-inning/group5.md [hunk 1]
```diff
@@ -1,33 +1,22 @@
...
-(Please include who is facilitating and how much time you are going to spend in each section):
-
-
-
-     Best practices:  No section should be longer than 20 minutes
-     Create a session that YOU would want to attend
-     Share resources in your slack channel
-     All sessions NEED to have a closing activity, reflection, discussion
+* 1. Introductions - meet your neighbor and introduce them to the class. (15 minutes)
+* 2. Intro to ei - Description and surface level explanation. (5 minutes)
```
Comment (emhickmann) [inline m4_sessions/1806-inning/group5.md:17]: Great--be clear on who is the lead for each of these sections.  Ideally build out a PPT for this and link it in your outline.

File: m4_sessions/1806-inning/group5.md [hunk 2]
```diff
@@ -1,33 +1,22 @@
...
+* 3. How ei can help with your work - Working in teams and developing empathetic apps. (20 minutes)
```
Comment (emhickmann) [inline m4_sessions/1806-inning/group5.md:18]: I would also connect this to hiring--people with strong EI can really shine in interviews, even if they don't have full technical background.

File: m4_sessions/1806-inning/group5.md [hunk 3]
```diff
@@ -1,33 +1,22 @@
...
+* 4. Read the article, class discussion (20 minutes)
```
Comment (emhickmann) [inline m4_sessions/1806-inning/group5.md:19]: Do you have a list of articles?  What are your discussion questions?

File: m4_sessions/1806-inning/group5.md [hunk 4]
```diff
@@ -1,33 +1,22 @@
...
+* 5. Watch Video and wrap up discussion (10 minutes)
```
Comment (emhickmann) [inline m4_sessions/1806-inning/group5.md:20]: Again, list of videos?  What are your discussion questions?

**Review by emhickmann — COMMENTED** [review]
@JackLaird0 This is a decent outline, but y'all have some serious work to do.  You also need to add a closing activity.

**Pull request merged**